cmd/flowrunner/main.go

```diff
@@ -31,6 +31,15 @@ type FlowTest struct {
 	Outputs      []json.RawMessage        `json:"outputs"`
 }
 
+func normalizeJSON(data json.RawMessage) ([]byte, error) {
```

Comment:
this works by loading the JSON into a map and then re-marshalling so that keys are serialized in alphabetical order